src/main/java/me/zeroeightsix/kami/util/StrafeUtils.kt

```diff
@@ -0,0 +1,19 @@
+package me.zeroeightsix.kami.util
+
+import net.minecraft.client.Minecraft
+import kotlin.math.*
+
+class StrafeUtils {
```

Comment:
You can make this as an object instead so you don't have to create a new instance every time you use it in a class

Reply:
😳